.github/workflows/documentation.yml

```
@@ -0,0 +1,25 @@
+name: Release
```

```suggestion
name: Documentation
```

We already have a workflow called Release